test/e2e/app.test.js

```diff
@@ -0,0 +1,86 @@
+const assert = require('assert')
```

Comment:
I did not really understand why this folder is called e2e, we are adding integration tests right? if so, this is misleading.

Reply:
Thanks for reviewing this PR.

You are probably right, but we discussed on slack about adding gRPC tests, and we got to the conclusion it would be better to add them at dcrwallet. Therefore, this tests were moved to https://github.com/decred/dcrwallet/pull/1298.

So this PR is abandoned, and I will close it.